keyboards/skeletonkbd/frost68/frost68.c

```diff
@@ -0,0 +1,100 @@
+}, {
+	{33.069956815567366, 3.682020999999999},
+	{45.669610362298535, 3.682020999999999},
+	{58.26926390902969, 2.3675390000000007},
+	{71.06408616238076, 6.4253908117277305},
+	{83.38840718545515, 10.386107811727733},
+	{95.71272820852954, 14.346825811727733},
+	{108.03704923160393, 18.307543811727733},
+	{99.25527325569391, 34.960896811727736},
+	{86.93095223261955, 31.000178811727736},
+	{74.60663120954516, 27.039460811727736},
+	{62.282310847869894, 23.078743811727733},
+	{49.20907990347908, 22.732020999999996},
+	{33.45951297006511, 22.732020999999996},
+	{32.34602106811951, 41.782021},
+	{49.67054469487486, 41.782021},
+	{62.74377563926569, 42.702634811727734},
+	{75.06809666234008, 46.663351811727736},
+	{87.39241702401533, 50.624069811727736},
+	{99.7167380470897, 54.584787811727736},
+	{103.2592830942541, 75.19885881172773},
+	{90.93496273257884, 71.23814081172773},
+	{78.61064170950445, 67.27742281172773},
+	{66.28632068643007, 63.31670481172773},
+	{53.27033449868614, 60.832021},
+	{32.795897485247984, 60.832021},
+	{28.071027405223802, 79.882021},
+	{46.97050772532055, 79.882021},
+	{80.61264662878455, 87.39640381172774},
+	{100.63966829128042, 93.83256981172772},
+	{140.16574696375307, 90.36694181172774},
+	{166.43688839595956, 82.53340481172773},
+	{195.65077951985486, 79.882021},
+	{211.40034645326884, 79.882021},
+	{224.0, 79.882021},
+	{236.5996535467312, 79.882021},
+	{224.0, 60.832021},
+	{206.67547637324466, 60.832021},
+	{189.3509527464893, 60.832021},
+	{176.7512991997581, 60.832021},
+	{163.73531367341332, 63.31670481172773},
+	{151.41099265033893, 67.27742281172773},
+	{139.08667162726456, 71.23814081172773},
+	{126.7623512655893, 75.19885881172773},
+	{130.3048963127537, 54.584787811727736},
+	{142.62921667442893, 50.624069811727736},
+	{154.95353769750332, 46.663351811727736},
+	{167.2778587205777, 42.702634811727734},
+	{180.35108966496855, 41.782021},
+	{192.95074321169966, 41.782021},
+	{213.42518022513784, 41.782021},
+	{236.59390069704355, 39.555001000000004},
+	{235.27110242442083, 20.505001},
+	{217.35607106638054, 22.732020999999996},
+	{201.6065041329666, 22.732020999999996},
+	{189.00685058623543, 22.732020999999996},
+	{176.40719703950424, 21.99143},
+	{163.39121151315945, 24.47611381172773},
+	{151.06689049008506, 28.43683181172773},
+	{138.74256946701064, 32.39754981172774},
+	{126.41824844393628, 36.358266811727745},
+	{128.14674563977667, 16.327184811727733},
+	{140.47106666285106, 12.366466811727733},
+	{152.79538768592545, 8.405748811727733},
+	{165.11970870899984, 4.4450318117277305},
+	{178.19293899199153, 2.941429999999997},
+	{190.7925925387227, 3.682020999999999},
+	{209.69207285881942, 3.682020999999999},
+	{233.94830415179814, 1.4550009999999958}
+},{
+	4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 8, 4,
+	4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+	4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+	4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+	4, 4, 4, 4, 4, 4, 4, 4
+} };
+// clang-format on
```

Comment:
RGB Matrix definition also belongs to `info.json` now, also:
a) You dont need such a precision when defining the positions
b) Those values are stored as `uint8_t` anyway, so the decimals are being stripped away 🙃 🤣

Reply:
Moved RGB Matrix layout information from `frost68.c` to `info.json`. And rounded off coordinates.
Some of the settings in `config.h` and `rules.mk` have also been moved to `info.json`.
a6357b0a4342e847aa2885affdf037653c1060d4